src/DataAccess/PredicatesDeserializer.php

```diff
@@ -22,7 +22,7 @@ public function deserialize( string $predicatesText ): PredicateList {
 		return new PredicateList(
 			array_values(
 				array_map(
-					fn( string $predicate ) => new Predicate( $predicate ),
+					static fn ( string $predicate ) => new Predicate( $predicate ),
```

Comment:
Is CS forcing this to be `static`? We don't normally do this.

Reply:
I've seen this being forced by CI before. By PHPCS IIRC.

Does not seem beneficial to force adding static here, so that is one of these rules that can be disabled. At the same time the code change itself does not hurt.